tests/Dockerfile

```diff
@@ -46,28 +46,44 @@ RUN dnf -y update && \
     dnf -y builddep /tmp/cockpit.spec && \
     dnf clean all
 
-RUN mkdir -p /usr/local/bin /home/user /build
-ADD cockpit-verify vm-prep network-cockpit.xml /usr/local/bin/
+RUN mkdir -p /usr/local/bin /home/user /build /build/tmp /secrets /images
+ADD cockpit-tests install-service vm-prep network-cockpit.xml /usr/local/bin/
 
-RUN chown -R user /build /home/user
 RUN usermod -a -G mock user
 RUN echo 'user ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
 RUN echo "config_opts['basedir'] = '/build/mock/'" >> /etc/mock/site-defaults.cfg
+RUN printf '[user]\n\t\nemail = cockpituous@cockpit-project.org\n\tname = Cockpituous\n' >/home/user/.gitconfig && \
+    ln -s /images /build/images && \
+    mkdir -p /home/user/.config /home/user/.ssh && \
+    ln -snf /secrets/ssh-config /home/user/.ssh/config && \
+    ln -snf /secrets/github-token /home/user/.config/github-token && \
+    git clone https://github.com/cockpit-project/cockpit /build/cockpit && \
+    cd /build/cockpit && npm install && \
+    chown -R user /build /secrets /images /home/user
```

Comment:
You can't chown `/secrets` any more as this is a  read-only mount now (that was the point of it). The chown needs to happen on the outside and is already done in `install-service`. I figure it might not actually fail as the files should already have the correct owner, but  I'd take it out for clarity.